proseco/catalog.py

```diff
@@ -153,6 +159,7 @@ def _get_aca_catalog(**kwargs):
         stars=aca.acqs.stars,
         fids=aca.fids,
         mons=aca.mons,
+        guides=guides,
```

Comment:
By passing the guides from get_guide_candidates to get_guide_catalog, we can skip doing the initial guide star selection *again* for this field (so this is here as an optimization).  However, the initial get_guide_catalog was run before the fids were selected so it may include candidate guide stars that are fid spoilers or fid trap spoilers.

Reply:
This needs an updated comment.